NoClassDefFoundError: net/bytebuddy/agent/ByteBuddyAgent

Trying to start it on Windows with Spigot 1.8.8 gives me NoClassDefFoundError for ByteBuddy

Error:
```
[05:31:29 ERROR]: Error occurred while enabling Slime v1.0-SNAPSHOT (Is it up to date?)
java.lang.NoClassDefFoundError: net/bytebuddy/agent/ByteBuddyAgent
        at me.hugmanrique.slime.bukkit.PluginClassInjector.inject(PluginClassInjector.java:41) ~[?:?]
        at me.hugmanrique.slime.bukkit.SlimePlugin.injectGenerator(SlimePlugin.java:52) ~[?:?]
        at me.hugmanrique.slime.bukkit.SlimePlugin.onEnable(SlimePlugin.java:22) ~[?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:321) ~[spigot-1.8.8-R0.1-SNAPSHOT-latest.jar:git-Spigot-db6de12-18fbb24]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:340) [spigot-1.8.8-R0.1-SNAPSHOT-latest.jar:git-Spigot-db6de12-18fbb24]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:405) [spigot-1.8.8-R0.1-SNAPSHOT-latest.jar:git-Spigot-db6de12-18fbb24]
        at org.bukkit.craftbukkit.v1_8_R3.CraftServer.loadPlugin(CraftServer.java:357) [spigot-1.8.8-R0.1-SNAPSHOT-latest.jar:git-Spigot-db6de12-18fbb24]
        at org.bukkit.craftbukkit.v1_8_R3.CraftServer.enablePlugins(CraftServer.java:317) [spigot-1.8.8-R0.1-SNAPSHOT-latest.jar:git-Spigot-db6de12-18fbb24]
        at net.minecraft.server.v1_8_R3.DedicatedServer.init(DedicatedServer.java:199) [spigot-1.8.8-R0.1-SNAPSHOT-latest.jar:git-Spigot-db6de12-18fbb24]
        at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:525) [spigot-1.8.8-R0.1-SNAPSHOT-latest.jar:git-Spigot-db6de12-18fbb24]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_191]
Caused by: java.lang.ClassNotFoundException: net.bytebuddy.agent.ByteBuddyAgent
        at java.net.URLClassLoader.findClass(Unknown Source) ~[?:1.8.0_191]
        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:101) ~[spigot-1.8.8-R0.1-SNAPSHOT-latest.jar:git-Spigot-db6de12-18fbb24]
        at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:86) ~[spigot-1.8.8-R0.1-SNAPSHOT-latest.jar:git-Spigot-db6de12-18fbb24]
        at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.8.0_191]
        at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.8.0_191]
        ... 11 more
```

Spigot:
```
This server is running CraftBukkit version git-Spigot-db6de12-18fbb24 (MC: 1.8.8) (Implementing API version 1.8.8-R0.1-SNAPSHOT)
```

--- a/bukkit/pom.xml
+++ b/bukkit/pom.xml
@@ -28,6 +28,7 @@
             <artifactId>slime-core</artifactId>
             <version>${project.parent.version}</version>
         </dependency>
+        <!-- Remember shaded dependencies must be included in shade includes -->
         <dependency>
             <groupId>net.bytebuddy</groupId>
             <artifactId>byte-buddy</artifactId>
@@ -84,6 +85,7 @@
                                 <includes>
                                     <include>me.hugmanrique:slime-core</include>
                                     <include>net.bytebuddy:byte-buddy</include>
+                                    <include>net.bytebuddy:byte-buddy-agent</include>
                                     <include>com.github.luben:zstd-jni:*:win_amd64</include>
                                 </includes>
                             </artifactSet>

